plone/app/caching/tests/test_utils.py

```diff
@@ -32,7 +31,7 @@ def stable_now():
     several tests compare this date with an Expires header,
     and this header may be set to ten years ago.
     """
-    tzinfo = pytz.timezone(TEST_TIMEZONE)
+    tzinfo = timezone(timedelta(hours=1), "Europe/Vienna")
     now = datetime.now() - timedelta(days=1000)
     now = tzinfo.localize(now)  # set tzinfo with correct DST offset
```

Comment:
```suggestion
    now = now.astimezone(tzinfo)  # set tzinfo with correct DST offset
```

@thet @davisagli this should, in my opinion, fix the tests here

Reply:
@mamico The suggestion makes sense (the localize method is unique to tzinfo from pytz).

But, a 1-hour offset for Europe/Vienna is only correct when summer time is not active. Does it matter here?

Reply:
I suggest creating a new branch of plone.app.caching to be used by Plone 6.0 and 6.1, and then make this change to use zoneinfo on the master branch (for Plone 6.2)